libs/providers/launchdarkly-client/README.md

```diff
@@ -0,0 +1,45 @@
+# launchdarkly-client Provider
+
+![Experimental](https://img.shields.io/badge/experimental-breaking%20changes%20allowed-yellow)
+
+An unofficial browser provider for LaunchDarkly.
+
+## Installation
+
+```
+$ npm install @openfeature/launchdarkly-client-provider
+```
+
+## Sample initialization
+``` ts
+  // init launchdarkly-js-client-sdk
+  const initialContext = {};
```

Comment:
Being as this context is an LDContext, it should contain either a `key` or `anonymous` should be true (in which case it will generate a key.)

Without something the SDK isn't going to initialize and will log
```
[LaunchDarkly] Invalid context specified. Please see https://docs.launchdarkly.com/sdk/client-side/javascript#initializing-the-client for instructions on SDK initialization.
```